tests/models/bert/test_modeling_tf_bert.py

```diff
@@ -606,6 +607,22 @@ class TFBertModelTest(TFModelTesterMixin, TFCoreModelTesterMixin, unittest.TestC
         if is_tf_available()
         else ()
     )
+
+    pipieline_model_mapping = (
+        {
+            "feature-extraction": TFBertModel,
+            "fill-mask": TFBertForMaskedLM,
+            "question-answering": TFBertForQuestionAnswering,
+            "text-classification": TFBertForSequenceClassification,
+            "text-generation": TFBertLMHeadModel,
+            "token-classification": TFBertForTokenClassification,
+            "zero-shot": TFBertForSequenceClassification,
+        }
+        if is_tf_available()
+        else {}
+    )
+    PipelineTesterMixin.disable_irrelevant_tests()
```

Comment:
This change LGTM, but as said above, I would remove the `disable_irrelevant_tests` method.